classes/submission/reviewRound/ReviewRound.php

```diff
@@ -312,6 +320,8 @@ public function getStatusKey($isAuthor = false)
                 return 'editor.submission.roundStatus.recommendationsReady';
             case self::REVIEW_ROUND_STATUS_RECOMMENDATIONS_COMPLETED:
                 return 'editor.submission.roundStatus.recommendationsCompleted';
+            case self::REVIEW_ROUND_STATUS_RETURNED_TO_REVIEW:
+                return 'editor.submission.roundStatus.returnToReview';
```

Comment:
As it's already completed action, let it be `editor.submission.roundStatus.returnedToReview`

Reply:
Done .